js/controller.js

```diff
@@ -268,11 +270,15 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
 
       this.accessibilityControls = new AccessibilityControls(this); //keyboard support
       this.state.screenToShow = CONSTANTS.SCREEN.INITIAL_SCREEN;
+
+      if (this.videoVr) {
+        this.mb.subscribe(OO.EVENTS.DIRECTION_CHANGED, 'customerUi', _.bind(this.setViewingDirection, this));
+      }
     },
 
     setVideoVr: function(event, obj) {
       this.videoVr = true;
-      this.videoVrSource = obj.source || {}; //if we need video vr params
+      this.videoVrSource = obj.source || false; //if we need video vr params
```

Comment:
Would null work better here? (if videoVrSource is intended to be an object)

Reply:
\+